tests/Cluster.py

```diff
@@ -93,13 +94,43 @@ def setChainStrategy(self, chainSyncStrategy=Utils.SyncReplayTag):
     def setWalletMgr(self, walletMgr):
         self.walletMgr=walletMgr
 
+    @staticmethod
+    def __defaultAlternateVersionLabels():
+        """Return a labels dictionary with just the "current" label to path set."""
+        labels={}
+        labels["current"]="./"
+        return labels
+
+    def setAlternateVersionLabels(self, file):
+        """From the provided file return a dictionary of labels to paths."""
```

Comment:
Do you need to check in a label file?

Reply:
No tests use it yet.  The one test I added the code so I could point @arhag to it so he could see how he can hook into it.